### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by abdozayan12**
Use React Router.
Use styles in a React app.

### CODE REVIEW

File: src/component/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit.
+        Quidem laudantium ad voluptatibus sit ab corrupti perferendis odio,
+        minus excepturi animi laborum corporis deserunt nesciunt non doloribus
+        quos possimus mollitia ea?
+      </p>
+    </div>
+  );
+}
+
```
Comment (Ibizugbe) [inline src/component/HomePage.js]: Nice job styling your project😉. Let's try to make sure our style structure sticks with the wireframe as this is a requirement in this milestone😉.

image

Take note of the highlighted elements and ensure they are positioned rightly in your project. also, take note of the spacings (margin and padding) and make the necessary change😉.

yours
image

image

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @abdozayan12 👋👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are passing👍.
- [x] Gitflow is used👍.
- [x] routes have been created using react-router👍.
- [x] Nav links for the 3 pages are displayed on each page👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 2]
```diff
@@ -1,6 +1,149 @@
...
+  height: 100vh;
+  padding: 20px;
+}
+
+.calc-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  min-width: 300px;
+  text-align: center;
+}
```
Comment (levy002) [inline src/App.css]: - [ ] The changes you made have affected the layout of the calculator page design. Kindly make sure to align the contents of the `Calculator` page and make sure to remove the horizontal scroll on this page in order to match the provided wireframe.

   - ### YOUR DESIGN
   chrome-capture-2023-6-11 (1)

   - ### PROVIDED WIREFRAME
   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @abdozayan12   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — APPROVED** [review]
Hi @abdozayan12 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Well written PR 👍 
* Good looking design 👍 
* Proper use of react router 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**